blogposts/announcing-sourcegraph-3.11.md

```diff
@@ -0,0 +1,204 @@
+
+[**🔎 Search query language statistics**](#search-query-language-statistics)<br />
+
+[**🗞 In other news**](#in-other-news)<br />
+How Lyft used Sourcegraph during its monolith to microservice decomposition and Sourcegraph proudly sponsoring GitLab Commit.
+
+[**📝 Changelog**](#changelog)<br />
+Every detail that changed in this release.
+
+[**🎖️ Thank you**](#thank-you)<br />
+Sourcegraph couldn't be what it is without the community.
+
+</div>
+
+**Deploy or upgrade:** [Local](https://docs.sourcegraph.com/#quickstart-guide) | [AWS](https://github.com/sourcegraph/deploy-sourcegraph-aws) | [DigitalOcean](https://marketplace.digitalocean.com/apps/sourcegraph?action=deploy&refcode=48dfb3ccb51c) | [Kubernetes cluster](https://github.com/sourcegraph/deploy-sourcegraph)
+
+## Introducing code-aware structural search
+
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/380662673?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/380662673" target="_blank">View on Vimeo</a></p>
+</p>
+
+Sourcegraph 3.11 introduces [structural search](https://docs.sourcegraph.com/user/search/structural), a code-aware search syntax that can identify structural patterns in code. This enables advanced code search to specifically match patterns inside code structures such as function parameters and loop bodies.
+
+It can be awkward or difficult to match code blocks or nested expressions with regexp. To meet this challenge we’ve introduced a new and easier way to search code that operates more closely on the parse tree of the input using [Comby syntax](https://comby.dev/) for structural matching.
+
+For example, this can be used to [find all the places that `fmt.Sprintf` are unnecessary](https://sourcegraph.com/search?q=repo:sourcegraph/sourcegraph%24+%27fmt.Sprintf%28%22:%5Bstr%5D%22%29%27&patternType=structural):
+
+```text
+patterntype:structural 'fmt.Sprintf(":[str]")'
+```
+
+Or to [identify all the places an `http.Client` is created with a `Transport`](https://sourcegraph.com/search?q=repo:sourcegraph/sourcegraph%24+%22http.Client%7B:%5Ba%5D+Transport:+:%5Bb%5D+:%5Bc%5D%7D%22+count:1000&patternType=structural):
+
+```text
+patterntype:structural "http.Client{:[a] Transport: :[b] :[c]}"
+```
+
+In these examples, the `:[str]` part is a hole with a descriptive name `str` that matches code. The important part is that these patterns understand that the structural pieces (e.g. parentheses and braces) are balanced.
+
+Known limitations:
+
+- Only indexed repositories will show results for structural search. On [Sourcegraph](https://sourcegraph.com/search), we index roughly the top 13,000 repositories by GitHub stars.
```

Comment:
```suggestion
- Only indexed repositories will show results for structural search. On [Sourcegraph](https://sourcegraph.com/search), we index approximately 10,000 of the most popular GitHub repositories.
```

Reply:
Ref: https://sourcegraph.slack.com/archives/C0W2E592M/p1574452433223800?thread_ts=1574451809.220400&cid=C0W2E592M

Reply:
I'll update the docs page to reflect this similarly in a follow up.